sequel/benchmarks/driver.rb

```diff
@@ -0,0 +1,194 @@
+#
+# Sequel Benchmark driver
+#
+require 'bundler/setup'
+require 'net/http'
+require 'json'
+require 'pathname'
+require 'optparse'
+require 'rails'
+require 'digest'
+
+RAW_URL = 'https://raw.githubusercontent.com/ruby-bench/ruby-bench-suite/master/sequel/benchmarks/'
+
+postgres_tcp_addr = ENV['POSTGRES_PORT_5432_TCP_ADDR'] || 'localhost'
+postgres_port = ENV['POSTGRES_PORT_5432_TCP_PORT'] || 5432
+mysql_tcp_addr = ENV['MYSQL_PORT_3306_TCP_ADDR'] || 'localhost'
+mysql_port = ENV['MYSQL_PORT_3306_TCP_PORT'] || 3306
+
+DATABASE_URLS = {
+  psql: "postgres://postgres@#{postgres_tcp_addr}:#{postgres_port}/rubybench",
+  mysql: "mysql2://root@#{mysql_tcp_addr}:#{mysql_port}/rubybench",
+}
+
+class BenchmarkDriver
+  def self.benchmark(options)
+    self.new(options).run
+  end
+
+  def initialize(options)
+    @repeat_count = options[:repeat_count]
+    @pattern = options[:pattern]
+  end
+
+  def run
+    files.each do |path|
+      next if !@pattern.empty? && /#{@pattern.join('|')}/ !~ File.basename(path)
+
+      if path.match(/sequel/)
```

Comment:
This driver is only responsible for running sequel benchmarks. Do we need this conditional statement?